content/en/blog/2024/otel-generative-ai/index.md

```diff
@@ -0,0 +1,200 @@
+---
+title: OpenTelemetry for Generative AI
+linkTitle: OpenTelemetry for Generative AI
+date: 2024-11-09
+author: >-
+  [Drew Robbins](https://github.com/drewby) (Microsoft),  [Liudmila
+  Molkova](https://github.com/lmolkova) (Microsoft)
+issue: [#5581](https://github.com/open-telemetry/opentelemetry.io/issues/5581)
```

Comment:
This is breaking the build I believe. Here's a fix:

```suggestion
issue: https://github.com/open-telemetry/opentelemetry.io/issues/5581
```

Reply:
thanks!